### PR TITLE
React update

### PR DESCRIPTION
**Pull request by sri2k1us**
- Update `react-gwt` to latest version
- Update to `webpack` 4.x
- Update to `React 16.7.0`
- Update `GWT` to `2.8.2`
- Update `storybook` (and related) for react to `4.1.7`
- Update `react-scripts` to latest version

And many more version updates. See `package.json` for full list.

### CODE REVIEW

**Comment by sri2k1us** [comment]
Note: Do NOT merge till before Tuesday Jan 29th Prod release

File: react-components/stories/util/hyperlink/DEHyperLink.stories.js [hunk 1]
```diff
@@ -16,3 +16,4 @@ class DEHyperLinkTest extends Component {
 }
 
 export default DEHyperLinkTest;
+                                                    0
```
Comment (psarando) [inline react-components/stories/util/hyperlink/DEHyperLink.stories.js]: Weird typo snuck in?

Comment (sri2k1us) [reply react-components/stories/util/hyperlink/DEHyperLink.stories.js]: 😕

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**